templates/markdown/.partials/param-prop.md

```diff
@@ -12,7 +12,13 @@
   </td>
   <td>{{prop.in}}</td>
   <td>{{{prop.descriptionAsHTML}}}</td>
-  <td>{{{acceptedValues prop.enum}}}</td>
+  <td>
+    {{#if prop.schema}}
+      {{{acceptedValues prop.schema.enum}}}
+    {{else}}
+      Any
+    {{/if}}
+  </td>
```

Comment:
You don't need to check for `prop.schema` first. Unlike javascript, this won't fail. The only change needed here, unless I'm missing something, is:

```mustache
<td>{{{acceptedValues prop.schema.enum}}}</td>
```